playroom/components.tsx

```diff
@@ -89,7 +88,7 @@ const useStyles = createUseStyles(() => ({
     },
 }));
 
-const themesMap: {[skinName in SkinName]: {themeConfig: ThemeConfig; text: string}} = {
+const themesMap: {[skinName: string]: {themeConfig: ThemeConfig; text: string}} = {
```

Comment:
TS was reporting this as an error, because O2-classic is missing in the themesMap object

Reply:
oh, there is a way to make this non-exhaustive?

Reply:
also, why don't we run tsc on CI? :/

Reply:
we are running tsc but it only checks files used by the library